cmd/node-termination-handler.go

```diff
@@ -242,6 +242,7 @@ func main() {
 					log.Info().
 						Str("event-id", event.EventID).
 						Str("kind", event.Kind).
+						Str("monitor", event.Monitor).
```

Comment:
Let's call this `aws-event-details` or perhaps `event-type-details`, and have it reflect the underlying AWS event. `kind` will continue to refer to the monitor type as it did in the past, to preserve backwards compatibility.

Reply:
I'd just call this `aws-event-type` or `aws-event-kind`. IMO adding `details` here is confusing because the field only contains one single detail.

Reply:
Agreed. Let's go with `aws-event-type`.